product/src/main/java/com/yas/product/service/ProductService.java

```diff
@@ -543,4 +543,32 @@ public List<ProductVariationGetVm> getProductVariationsByParentId(Long id) {
             throw new BadRequestException(Constants.ERROR_CODE.PRODUCT_NOT_HAVE_VARIATION, id);
         }
     }
+
+    public List<ProductExportingDetailVm> exportProducts(String productName, String brandName) {
+        List<Product> productList = productRepository.getExportingProducts(productName.trim().toLowerCase(),
+                brandName.trim());
+
+        return productList.stream()
+                .map(product -> new ProductExportingDetailVm(product.getId(),
+                        product.getName(),
+                        product.getShortDescription(),
+                        product.getDescription(),
+                        product.getSpecification(),
+                        product.getSku(),
+                        product.getGtin(),
+                        product.getSlug(),
+                        product.getIsAllowedToOrder(),
+                        product.getIsPublished(),
+                        product.getIsFeatured(),
+                        product.getIsVisibleIndividually(),
```

Comment:
`product.getIsXXX` will return Boolean value, which is a wrapper type. A Boolean wrapper type can hold null value. Please recheck that if we accept null value for this column in database. Otherwise, feel free to ignore this comment.

Reply:
I agree with you but according to bussiness we only export products with isVisibleIndividually = TRUE so I think it is not necessary to check.Thanks